### PR TITLE
Enhance debugability for kubernetes rig

### PR DESCRIPTION
**Pull request by apostasie**
Breakout of #3535 

Silence pull noise, and leftover cleanup.

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]
Thanks

File: hack/build-integration-kubernetes.sh [hunk 1]
```diff
@@ -104,8 +104,10 @@ main(){
 
   # Hack to get go into kind control plane
   exec::nerdctl rm -f go-kind 2>/dev/null || true
-  exec::nerdctl run -d --name go-kind golang:"$GO_VERSION" sleep Inf
+  exec::nerdctl pull --quiet golang:"$GO_VERSION"
+  exec::nerdctl run -d --pull never --name go-kind golang:"$GO_VERSION" sleep Inf
```
Comment (AkihiroSuda) [inline hack/build-integration-kubernetes.sh:108]: I think `nerdctl run` now has `--quiet` too

Comment (apostasie) [reply hack/build-integration-kubernetes.sh:108]: Oh. Let me check.

Comment (apostasie) [reply hack/build-integration-kubernetes.sh:108]: You are right. Will amend.

Comment (apostasie) [reply hack/build-integration-kubernetes.sh:108]: Done.

Comment (AkihiroSuda) [reply hack/build-integration-kubernetes.sh:108]: Probably you forgot to push the amended commit

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**